packages/api/src/services/token.js

```diff
@@ -0,0 +1,24 @@
+const jwt = require('jsonwebtoken');
+
+function generate(payload, expiresIn = '1d') {
+  return new Promise((resolve, reject) => {
+    jwt.sign(payload, process.env.JWT_SECRET, { expiresIn }, (err, token) => {
```

Comment:
Get the secret in a `config` file instead of calling `process.env` multiple times. Though, it may not be that bad in simple application. However, this routing will be called multiple times and since we don't expect the SECRET to change, we can cache it in the application layer. 

Read [this](https://github.com/nodejs/node/issues/3104).